src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -471,6 +471,8 @@ public override bool CanApplyChange(ApplyChangesKind feature)
                 case ApplyChangesKind.AddAnalyzerConfigDocument:
                 case ApplyChangesKind.RemoveAnalyzerConfigDocument:
                 case ApplyChangesKind.ChangeAnalyzerConfigDocument:
+                case ApplyChangesKind.AddSolutionAnalyzerReference:
+                case ApplyChangesKind.RemoveSolutionAnalyzerReference:
```

Comment:
What does this mean to allow such a change? We're letting extensions just come along and hook up additional ones dynamically?

Reply:
Yes. These can be added/removed dynamically now.